modules/dataquery/help/DQThelp.md

```diff
@@ -0,0 +1,47 @@
The “Define Fields” tab is where you select the data categories and the specific data fields that you want added to your query. By default, all categories of data fields are displayed in a list when you first enter the tab. If you click on any category, its field names and field descriptions will appear in a table. If the number of fields in the chosen category exceeds the display limit per page, the results may be displayed on subsequent pages within the table, accessible via the page number buttons found by scrolling toward the bottom of the page. Any fields that are defined for the current query are listed on the right-hand side (subject to change). If there are no fields defined for the current query, this list is not shown. Time points can be specified within the field in the “Define Fields” tab or in the “Define Filters” tab. The options to add fields to your query are as follows: Option 1: Add Using Dropdown Fields can be searched for using the list of categories of data fields. After clicking the desired instrument, a list of fields are displayed. Clicking on a field row will highlight it and add it to the list of selected fields for your current query. Option 2: Search Within Instrument Fields can be searched for using the search bar. Once the user starts typing, all matches will be displayed in an active list. Once the desired field is displayed, the field must be clicked to be added into the query. Option 3: Add All If all fields from a particular category should be included in the query, use this method. By clicking the “Add All” button above the list of categories, all fields from the category will be added. Option 4: Load a Saved Query To reload a saved query, click on the "Load Saved Query" tab. User Saved Queries and Shared Saved Queries will appear in a dropdown. User Saved Queries are queries that the current user has saved, and are listed according to Query Name. Shared Saved Queries are publicly shared, and are listed according to the user who created it and the Query Name. Once a query is selected, the DQT will retrieve and load the query fields and filters that were selected when the query was initially created. This will be displayed under the “Define Fields” and “Define Filters” tabs, respectively. If you wish to remove fields from your query, you must find the field from the list (using the Search Within Instrument search bar or in the instrument list of fields) and click on the highlighted field. This can be confirmed by checking that the field no longer appears in your list of selected fields. If you wish to remove all currently selected fields from your query, you can click “Clear Query” in the list of Fields. If you wish to remove all fields from a specific category, click the “Remove All” button above the list of categories. Once all desired fields are listed under “Fields”, continue to the “Define Filters” tab to add filters to your query, or go directly to the “View Data” tab if your query does not require filters.
```

Comment:
would it be possible to add a little bit of formatting in this paragraph? like a new line for each of the options.

Reply:
@PapillonMcGill Melanie great idea - I've tried a few things that might help in my latest commit on this branch -- could you kindly re-test ? 

When testing -- kindly paste below a screenshot (of your Help dropdown) showing @knoll-alex  and I what you see for the "Define Fields" section - merci!